core/src/scss/components/hero/_hero--caption.scss

```diff
@@ -0,0 +1,45 @@
+@charset "UTF-8";
+
+.su-hero--caption {
+
+  // Ensure no card shows up.
+  .su-hero__card {
+    display: none;
+  }
+
+  // The media element shall not be absolute in this option.
+  .su-hero__media {
+    @include grid-media('md') {
+      max-height: map-get($_su-hero-height, 'md');
+      position: relative;
+    }
+
+    @include grid-media('lg') {
+      max-height: map-get($_su-hero-height, 'lg');
+      position: relative;
+    }
+
+    @include grid-media('xl') {
+      max-height: map-get($_su-hero-height, 'xl');
+      position: relative;
+    }
+  }
+
+  .su-hero__content {
+    @include centered-column;
+    @include modular-spacing(padding-top, 0);
+    @include modular-spacing(padding-bottom, 2);
+    text-align: right;
+    display: block;
+
+    p {
+      @include credits;
+    }
+
+    // Only allow for 60% width on large screen sizes.
+    @include grid-media('lg') {
+      @include padding(null null null 40%);
```

Comment:
This 40% padding-left looks larger than 40% because it's 40% of the full browser width (including the screen edges) instead of 40% of the center container width. If the width doesn't have to be exactly 60% then it would probably look close enough with padding-left:30%. This issue is more obvious when the browser is really wide (try resizing to 2000px, the caption gets quite narrow).

Reply:
Ah yeah, good catch. I was thinking since it was full width but it really isn't. I have revised.